tests/modules/programs/helix/example-settings.nix

```diff
@@ -21,10 +21,22 @@ with lib;
         };
       };
 
-      languages = [{
-        name = "rust";
-        auto-format = false;
-      }];
+      languages = {
+        language-server.typescript-language-server = {
+          # depending on derivation outputs will change the generated settings everytime nixpkgs changes...
```

Comment:
We use [package stubs](https://github.com/nix-community/home-manager/blob/master/tests/stubs.nix) for this; you can look at the other tests for inspiration

Reply:
Thanks for the tip, fixed it in the latest force-push